scss/_forms.scss

```diff
@@ -211,64 +210,105 @@ dl.form {
     }
   }
 
+  .form-note {
```

Comment:
You know, I have no idea. This isn't actually an addition, it just moved as a result of pasting stuff in. This diff isn't entirely accurate in that regard... Unless I'm mistaken, it's in `master`. Should I kill it?